tiles/src/test/java/com/protomaps/basemap/names/ScriptTest.java

```diff
@@ -0,0 +1,85 @@
+package com.protomaps.basemap.names;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+class ScriptTest {
+
+  @Test
+  void singleScript() {
+    assertEquals("LATIN", Script.getScript("Berlin"));
+    assertEquals("ARABIC", Script.getScript("دبي"));
+    assertEquals("CYRILLIC", Script.getScript("Скопје"));
+    assertEquals("HAN", Script.getScript("北京"));
```

Comment:
We should try to be explicate about Simplified Chinese versus Traditional Chinese

Reply:
And add a comment this is {Simplified|Traditional} "Chinese, please.

Reply:
I think unicode actually uses the same codepoints for those, see https://en.wikipedia.org/wiki/Han_unification. I could not find any counter examples but I am happy to be proven wrong. Added a code comment for now...